tests/testthat/test-bleach.R

```diff
@@ -47,10 +47,10 @@ test_that("bleach_layer() works", {
     geom_line(aes(colour = NULL, fill = NULL), d, colour = grey07)
   )
 
-  # If the geom accepts fill, it is sets to grey even when it is not included in the mapping.
+  # If the geom accepts fill, it is set to grey even when it is not included in the mapping.
   expect_equal_layer(
     bleach_layer(geom_col(aes(colour = type), d), g_info, list()),
-    geom_col(aes_bleached, d_bleached, colour = grey07, fill = grey07)
+    geom_col(aes_bleached, d_bleached, colour = NA, fill = grey07)
```

Comment:
Honestly, I'm not sure why I wrote the expectation as non-NA, so I'm no longer sure what value this is supposed to be...